kernel/kernel-api/src/main/java/io/delta/kernel/ScanBuilder.java

```diff
@@ -65,4 +67,14 @@ public interface ScanBuilder {
 
   /** @return Build the {@link Scan instance} */
   Scan build();
+
+  /**
+   * Build a Paginated Scan with a required page size and an optional page token.
+   *
+   * @param pageSize Maximum number of Scan Files to return in this page.
+   * @param pageToken Optional page token representing the current scan position; empty to start
```

Comment:
nit: start position?

Reply:
`empty to start from the beginning` --> `to retrieve the first page in a paginated scan, pass in an {@link Optional#empty}`